influxql/README.md

```diff
@@ -98,16 +98,16 @@ _cpu_stats
 ALL           ALTER         ANY           AS            ASC           BEGIN
 BY            CREATE        CONTINUOUS    DATABASE      DATABASES     DEFAULT
 DELETE        DESC          DESTINATIONS  DIAGNOSTICS   DISTINCT      DROP
-DURATION      END           EVERY         EXISTS        EXPLAIN       FIELD
-FOR           FORCE         FROM          GRANT         GRANTS        GROUP
-GROUPS        IF            IN            INF           INNER         INSERT
-INTO          KEY           KEYS          LIMIT         SHOW          MEASUREMENT
-MEASUREMENTS  NAME          NOT           OFFSET        ON            ORDER
-PASSWORD      POLICY        POLICIES      PRIVILEGES    QUERIES       QUERY
-READ          REPLICATION   RESAMPLE      RETENTION     REVOKE        SELECT
-SERIES        SET           SHARD         SHARDS        SLIMIT        SOFFSET
-STATS         SUBSCRIPTION  SUBSCRIPTIONS TAG           TO            USER
-USERS         VALUES        WHERE         WITH          WRITE
+DURATION      END           EVERY         EXPLAIN       FIELD         FOR
+FORCE         FROM          GRANT         GRANTS        GROUP         GROUPS
+IF            IN            INF           INNER         INSERT        INTO
```

Comment:
Can't we remove `IF` too? I could be wrong, but I'm not aware of it being used outside of `IF EXISTS` and `IF NOT EXISTS`?